frontend/frontend-react/src/main/App.css

```diff
@@ -10,7 +10,7 @@
 }
 
 body {
-    height: 100vh;
+    height: auto;
     width: 100vw;
```

Comment:
```suggestion
    width: 100%;
```
I think you should use `width: 100%` instead of `width: 100vw` because `100vw` tries to occupy the whole screen but when the screen has a scroll it created an `overflow-x`. Example:
![Screenshot from 2021-06-23 09-23-19](https://user-images.githubusercontent.com/54550926/123097388-7d2e2300-d406-11eb-8bb5-513fc4a4e8eb.png)